_maps/map_files/generic/CentCom.dmm

```diff
@@ -5063,10 +5060,8 @@
 /turf/open/floor/mineral/titanium/tiled/yellow,
 /area/centcom/syndicate_mothership/expansion_bombthreat)
 "anB" = (
-/obj/structure/filingcabinet/filingcabinet,
-/obj/effect/turf_decal/tile/neutral/fourcorners,
-/turf/open/floor/iron/dark,
-/area/centcom/central_command_areas/ferry)
+/turf/open/space/basic,
```

Comment:
```suggestion
/turf/cordon,
```
The cordon area alone won't fix all the issues - you need to use the cordon turf too.